neo/rawio/openephysbinaryrawio.py

```diff
@@ -431,6 +461,8 @@ def _channels_to_group_id(self, channel_indexes):
         return group_id
 
     def _get_signal_t_start(self, block_index, seg_index, stream_index):
+        if stream_index >= len(self._sig_streams[block_index][seg_index]):
+            stream_index = stream_index - len(self._sig_streams[block_index][seg_index])    
         t_start = self._sig_streams[block_index][seg_index][stream_index]["t_start"]
```

Comment:
I understand correctly the ptach `stream_index` is not anymore a stream_index but an internal variable which is not a direct mapping to stream_index.
`self._sig_streams[block_index][seg_index][stream_index]["t_start"]`

If this is the case I would change this internal variable to something else to avoid mistale when reading the entire code.

Maybe I am mistunderstanding or not clear at all but this writting is a bit weird to me.

Reply:
You are correct. This is a good idea.

Reply:
Changed.